org.eclipse.xtend.lib.gwt.test/.settings/org.eclipse.core.resources.prefs

```diff
@@ -1,2 +1,3 @@
 eclipse.preferences.version=1
 encoding/<project>=ISO-8859-1
+encoding/src=ISO-8859-1
```

Comment:
Curiosity: Isn't this redundant?

Reply:
Looks like m2e likes to add that information for source folders when you update the project.